src/main/java/org/fundacion/jala/converter/models/User.java

```diff
@@ -59,104 +59,117 @@ public User(final String newName, final String newPassword, final String newToke
     }
 
     /**
-     * Obtains Ids of the User
-     * @return id of the user
+     * Obtains Ids of the User.
```

Comment:
The user id should be unique, so it should not be plural. Maybe, it will be better to put:
Obtains user's id

Reply:
ok